Assets/GothicVR/Scripts/Bootstrap/PhoenixBootstrapper.cs

```diff
@@ -115,6 +115,11 @@ private void LoadMusic()
             Debug.Log("Loading music");
         }
 
+        private void LoadTextures()
+        {
+            UIManager.I.LoadDefaultTextures();
```

Comment:
As I said before: If we load Loading.unity scene after bootstrapping is done (Which I prefer), then we can remove this one here and have it inside Start() of the Manager on the Loading scene.